### PR TITLE
index imports for models (#5654)

### LINKED ISSUE #5654: [FEATURE] Imports to have an index file
### Description

We now have aliases for imports, for example...

```js
import ModelA from "@models/..."
import ModelB from "@models/..."
import ModelC from "@models/..."
```

but we need to have a line for each imported model file.

It would be cleaner to have an `index.js` file in each alias import folder, so we can do...

```js
import { ModelA, ModelB, ModelC } from "@models/..."
```

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Shaikh-Danish**
Closes #5654

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6147"></a>

### CODE REVIEW

**Comment by amandamartin-dev** [comment]
Hello @Shaikh-Danish there are some conflicts to resolve in one file.  Please take a look and let us know if you need any help prior to review

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for your contribution.

There are failing tests. Also when I try to run the code, I get a lot of errors that the models are not working now.

File: models/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import Link from './Link'
+import LinkStats from './LinkStats'
+import Profile from './Profile'
+import ProfileStats from './ProfileStats'
+import Session from './Session'
+import Stats from './Stats'
+import User from './User'
+import VerificationToken from './VerificationToken'
+
+export default {
```
Comment (eddiejaoude) [inline models/index.js:11]: This should not be `default`, this is what is causing the app to no longer work

**Comment by eddiejaoude** [comment]
I will fix via a temporary branch

File: pages/api/discover/random.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import connectMongo from "@config/mongo";
 import logger from "@config/logger";
-import Profile from "@models/Profile";
+import loadProfiles from "@services/profiles/loadProfiles";
```
Comment (eddiejaoude) [inline pages/api/discover/random.js:3]: Not sure why this import was added? It is not used in the file, maybe from a bad merge (or git conflict)

**Pull request merged**